hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3515,9 +3519,18 @@ public S3VolumeContext getS3VolumeContext() throws IOException {
       }
     }
 
-    // getVolumeInfo() performs acl checks and checks volume existence.
+
+    final OmVolumeArgs volumeInfo;
+    if (internal) {
```

Comment:
A minor nit: can we consider renaming `internal` to something more descriptive, like `skipChecks`?